Isosurfacing_3/include/CGAL/Isosurfacing_3/internal/partition_traits_Octree.h

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2022-2024 INRIA Sophia-Antipolis (France), GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Julian Stahl
+//                 Mael Rouxel-Labbé
+
+#ifndef CGAL_ISOSURFACING_3_INTERNAL_PARTITION_TRAITS_OCTREE_H
+#define CGAL_ISOSURFACING_3_INTERNAL_PARTITION_TRAITS_OCTREE_H
+
+#include <CGAL/license/Isosurfacing_3.h>
+
+#include <CGAL/Isosurfacing_3/internal/Octree_wrapper.h>
+#include <CGAL/Isosurfacing_3/internal/Cell_type.h>
+
+#include <CGAL/tags.h>
+
+#ifdef CGAL_LINKED_WITH_TBB
+#include <tbb/parallel_for.h>
+#endif // CGAL_LINKED_WITH_TBB
+
+#include <array>
+
+namespace CGAL {
+namespace Isosurfacing {
+namespace internal {
+
+template <typename GeomTraits>
+class Octree_wrapper;
+
+} // namespace internal
+
+template <typename Partition>
+struct partition_traits;
+
+template <typename GeomTraits>
+struct partition_traits<internal::Octree_wrapper<GeomTraits> >
+{
+  using Octree = internal::Octree_wrapper<GeomTraits>;
+
+public:
+  using vertex_descriptor = typename Octree::Vertex_handle;
+  using edge_descriptor = typename Octree::Edge_handle;
+  using cell_descriptor = typename Octree::Voxel_handle;
+
+  static constexpr Cell_type CELL_TYPE = CUBICAL_CELL;
+  static constexpr std::size_t VERTICES_PER_CELL = 8;
+  static constexpr std::size_t EDGES_PER_CELL = 12;
+
+  using Edge_vertices = std::array<vertex_descriptor, 2>;
```

Comment:
`Edge_vertices` here is called `[Vertices_incident_to_edge](https://cgal.geometryfactory.com/~mrouxell/Isosurfacing_3/V3/Isosurfacing_3/classCGAL_1_1Isosurfacing_1_1partition__traits.html#ae9f271a4c42a8a78401e47620eac7630)` in the concept

Reply:
You had already found the problem in your R3 review (https://github.com/CGAL/cgal/pull/6849#discussion_r1526386969), it was fixed in https://github.com/CGAL/cgal/commit/2441f9183a21e42cc21f649d2afaa5356ba4541f.